### DISCUSSION TITLE
Why the texture of the gem tools was changed?

### DISCUSSION THREAD

**Discussion by TerrariaMinecraftFan44** [MinicraftPlus/minicraft-plus-revived#657] [Q&A] [answered]
**Why the gem tools texture was changed?**
Why the texture of the gem tools was changed?, I know gems are purple, but the blue texture of the gem tools was unique and better (in my opinion and some of my friends opinion). Being like Minecraft diamond tools made gem tools unique in color.

**Comment by TerrariaMinecraftFan44** [comment]
And also, why the Nether update was changed for the Obsidian Knight?

**Comment by Litorom** [comment, marked as answer]
1. Blue Gem tools that were made with Purple Gems makes no sense. Also this isn’t Minecraft, it’s Minicraft, which is different. This isn’t a 2D replica of Minecraft and never was with the original by Markus Persson.
2. There was no Nether Update and never was, **idk where you got that information. The Obsidian Dungeon isn’t the nether and was an outdated feature. A boss in the obsidian dungeon was planned by the original maintainers anyway.

** I found it, please review the comment I added.

**Comment by TerrariaMinecraftFan44** [comment]
Ok I understand, thank you for answering me

**Comment by BenCheung0422** [comment]
Should we keep questions opened in case of duplicated questions?
